lib/secure_headers/railtie.rb

```diff
@@ -3,9 +3,17 @@
   module SecureHeaders
     class Railtie < Rails::Engine
       isolate_namespace ::SecureHeaders if defined? isolate_namespace # rails 3.0
+      conflicting_headers = ['X-Frame-Options', 'X-XSS-Protection', 'X-Content-Type-Options']
```

Comment:
I think we should future proof this by including the headers this gem sets, even if they aren't in default headers.

i.e. csp, csp report only, hpkp, xdownload, etc